server/src/main/java/com/objectcomputing/checkins/services/skills/SkillRepository.java

```diff
@@ -17,4 +20,11 @@
 
     List<Skill> findByPending(boolean pending);
 
+    @Override
+    @Join(value = "tags", type = Join.Type.LEFT_FETCH)
```

Comment:
Does this have an effect with JDBC repos?

Reply:
Yes. If this isn't here JDBC won't do a fetch on the joined records and the skill will be returned without its tags.